src/Frame.js

```diff
@@ -0,0 +1,26 @@
+function Frame(){
+  this.rollCount = 1;
+  this._over = false;
+  this.isStrike = false;
+  this.isSpare = false;
+  this._pinsLeft = 10;
+};
+
+Frame.prototype.roll = function(pins){
+  this._pinsLeft -= pins;
+  if (this.rollCount === 1 && this._pinsLeft === 0) {
+    this._over = true;
+    this.isStrike = true;
+  } else if (this.rollCount === 1 && this._pinsLeft !== 0){
+    this.rollCount = 2;
+  } else if (this.rollCount === 2 && this._pinsLeft === 0) {
+    this._over = true;
+    this.isSpare = true;
+  } else {
+    this._over = true;
+  };
```

Comment:
Really like the layout of Frame, it's super clear and easy to understand. The names reflect quite well what the functions are in charge of, and your if statement is easy to follow! :)